test-app/src/test/resources/application-test.yml

```diff
@@ -13,11 +13,15 @@ hmpps.sqs:
       queueName: ${random.uuid}
       subscribeTopicId: inboundtopic
       subscribeFilter: '{"eventType":[ "OFFENDER_MOVEMENT-RECEPTION", "OFFENDER_MOVEMENT-DISCHARGE", "test.type"] }'
+      asyncDlqClient: true
```

Comment:
these have to be async to get the listeners registered

Reply:
Maybe we could put some validation in `HmppsSqsProperties` such that we fail if there is a subscription without an async client.